src/content.js

```diff
@@ -47,20 +47,22 @@ Http.onreadystatechange=(e)=> {
   if (!Http.responseText || Http.status !== 200 || Http.readyState !== 4 ) {return}
   const text = JSON.parse(Http.responseText);
   const emoji = text.emoji;
-  for(let key in emoji) {
-    if (key.includes('alias:')) {
-      key = key.replace('alias:', '');
+  for (let key in emoji) {
```

Comment:
delete this <img src="https://emoji.slack-edge.com/T02AKE45B/editdelete/6ed9be86c4cf2edb.png" height="28">